newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js

```diff
@@ -275,7 +412,12 @@ const ImagePreview = ({
             {!hideControls && (
               <MiniToolbar noPadding>
                 <IconButton
-                  onClick={() => zoomBy(zoomOutFactor)}
+                  onClick={() =>
+                    zoomAroundPointBy(zoomOutFactor, [
+                      (containerWidth || 0) / 2,
+                      (containerHeight || 0) / 2,
```

Comment:
took me a while to understand that you're zooming around the current frame center (and not the whole image center)
But it's great :)